hiring-developers-with-readthedocs.rst

```diff
@@ -0,0 +1,83 @@
+.. post:: October 30, 2018
+   :tags: advertising, sustainability, hiring
+   :author: David
+   :location: SAN
+
+
+Tips to Hire Developers with Read the Docs
+==========================================
+
+Read the Docs is probably not the first place you think of if you are recruiting.
+However, over **7 million unique developers use Read the Docs each month** from all over the world.
+We didn't set out to build a better job board,
+but after a number of advertisers used our `ethical ads`_ for recruiting,
+we discovered that Read the Docs was a great place to find developer talent.
+
+.. _ethical ads: https://docs.readthedocs.io/en/latest/advertising/ethical-advertising.html
+
+
+Hiring with Read the Docs works 
+-------------------------------
+
+Developers are not always actively job seeking
+by browsing job boards or company careers pages.
+However, they are on Read the Docs reading about the libraries and frameworks they use.
+Even when people aren't actively looking for a new job,
+many are open to exploring new opportunities.
```

Comment:
not my field but shouldn't be "to explore" here?

Reply:
I don't believe there is a grammatical issue with the last sentence ending "...  many are open to exploring new opportunities."